### ISSUE TITLE
maskrcnn的检测结果过低

### ISSUE THREAD

**Issue by 6098669** [open-mmlab/mmocr#1665]
### Model/Dataset/Scheduler description

在ctw1500上的结果才74，至少也得80以上吧

### Open source status

- [ ] The model implementation is available
- [ ] The model weights are available.

**Comment by Mountchicken** [comment]
Hi @6098669 
The implementation of MaskRCNN in MMOCR only uses 160 epochs for training, which is much less than other text detection algorithms. Using more epochs for training may achieve higher results.

**Comment by 6098669** [comment]
> 

训练次数少不是理由，同类型的模型contournet也没训练多少个epoch，f值都80多

**Comment by gaotongxiao** [comment]
Using MaskRCNN as a text detector mainly demonstrates the capability of directly using MMDetection's model, but the training hyperparameters are not necessarily optimum. We appreciate any insights on improving its performance.
